src/einsteinpy/plotting/mplplotter.py

```diff
@@ -0,0 +1,89 @@
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.patches import Circle
+
+
+class MatplotlibPlotter:
```

Comment:
Maybe name the class as StaticGeodesicPlotter?  :P

Reply:
Should I remove the old ones then?

Reply:
No! Put them in a separate folder. Name that folder, "senile" :P

Reply:
And anyways, put this all in geodesics folder :)